account_payment_term_discount/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2010-2012 Camptocamp Austria (<http://www.camptocamp.at>)
+# Copyright (C) 2018 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Account Payment Terms Discount",
+    "version": "14.0.1.1.0",
+    "license": "AGPL-3",
+    "category": "Accounting & Finance",
+    "depends": ["purchase", "account"],
```

Comment:
I see `purchase` as a forced dependency an issue.
This forces customers to pay for that additional module, even if they don't need it.
Can this dependency be split to a separate module?